packages/thunderstore-api/src/apiFetch.ts

```diff
@@ -39,32 +42,44 @@ async function fetchRetry(
   }
 }
 
+/**
+ * Simple timeout helper used by the retry loop.
+ */
 function sleep(delay: number) {
   return new Promise((resolve) => setTimeout(resolve, delay));
 }
 
-export type apiFetchArgs<B, QP> = {
+/**
```

Comment:
This comment is actively confusing me. It seems it would be more suited for `apiFetchArgs` than `SchemaOrUndefined`. And if it would be a comment for `apiFetchArgs` I would nitpick that it provides no value.